src/pages/achievements/[id].tsx

```diff
@@ -96,14 +99,38 @@ const Achievement = () => {
     return user.id === achievement?.user_id;
   }, [user, achievement]);
 
-  const mintable = useMemo(() => {
-    return achievement?.community?.can_mint_certificates;
+  const isICPSubmission = useMemo(() => {
+    return achievement?.community.name === "Internet Computer";
   }, [achievement]);
 
+  const mintable = useMemo(() => {
+    return achievement?.community?.can_mint_certificates || isICPSubmission;
+  }, [achievement, isICPSubmission]);
+
   const isNotCertificateIcon = useMemo(() => {
     return !achievement?.metadata?.image?.includes("/img/certificates/");
   }, [achievement]);
 
+  const addCourseCompletionToTheIssuerCanister = useCallback(async () => {
+    if (!achievement?.metadata) return;
+    const { name } = achievement?.metadata;
+    if (!window.issuerCanister) return;
+    try {
+      setAddCourseToCompletionPending(true);
+      await window.issuerCanister.add_course_completion(name.toLowerCase().replaceAll(" ", "-"));
+      await dispatch(completeIcpCertificate({ id: achievement.id }));
```

Comment:
so if this api endpoint gets triggered certificate be marked as completed even if icp hasn't been triggered

Reply:
Yes, but the badge can never be verified with badge gate or any other, as badge gate does not communicate with dacade backend.